department-of-veterans-affairs/abd-vro
# Feature/mcp 1962 disable endpoints

# Removes two end points from qa+ environments

## Description



### What was the problem?

Unused endpoints were going to be scanned in qa environments. These end points are useful in development so they are left  for dev and default environments.



### How does this fix it?

Endpoints disables using Profiles. Disabled end points are
1) health-data-access
2) fetch-claims

Also cleaned up descriptions since this open api doc will be sent to scanners. 

### Jira Tickets



- [MCP-1962](https://amida.atlassian.net/browse/MCP-1962)

## How to test this PR

- Verify end points are working as before in dev environment
- Set ENV = qa ad build
- Verify that you can not curl to the disabled end points.

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
